sygnal/gcmpushkin.py

```diff
@@ -134,14 +144,16 @@ def __init__(self, name, sygnal, config):
         # Use the fcm_options config dictionary as a foundation for the body;
         # this lets the Sygnal admin choose custom FCM options
         # (e.g. content_available).
-        self.base_request_body: dict = self.get_config("fcm_options", {})
+        self.base_request_body: dict = cast(dict, self.get_config("fcm_options", {}))
         if not isinstance(self.base_request_body, dict):
             raise PushkinSetupException(
                 "Config field fcm_options, if set, must be a dictionary of options"
             )
```

Comment:
again, this is awkward (and it's already being checked) but using `cast` should be avoided
```suggestion
        base_request_body_raw = self.get_config("fcm_options", {})
        if not isinstance(base_request_body_raw, dict):
            raise PushkinSetupException(
                "Config field fcm_options, if set, must be a dictionary of options"
            )
        self.base_request_body = base_request_body_raw
```